README.md

```diff
@@ -21,16 +21,20 @@ HerdDB uses **Apache Calcite** as SQL parser and SQL Planner
 
 ## Basic concepts
 
-Data, as in any **SQL database**, is organized in tables and, in order to leverage HerdDB replication function, tables are grouped inside **Tablespaces**.
+Data, as in any **SQL database**, is organized in tables and tables are grouped inside **Tablespaces**.
 
-A Tablespace is a logical set of tables that is the fundamental architectural brick upon which the replication is built.
-
-There are some DB features which are available only among tables of the same tablespace:
+A Tablespace is the fundamental architectural brick upon which the replication is built and some DB features are available only among tables of the same tablespace:
 - transactions may span only tables of the same tablespace
 - subqueries may span only tables of the same tablespace
 
-Replication is configured at tablespace level, so for each tablespace only one server is designed to be the 'leader' (manager) and then you may configure a set of 'replicas'.
-The system automatically replicates data between replicas and handles transparently server failures.
+Replication is configured at tablespace level, so for each tablespace only one server is designed to be the 'leader' (manager) at a specific point in time, then you may configure a set of 'replicas'.
+The system automatically replicates data between replicas and handles server failures transparently.
+
+## Overview
+
+Intoducing HerdDB - Pulsar Summit 2020 - Enrico Olivelli
+
+[![Intoducing HerdDB - Youtube link](https://img.youtube.com/vi/K7xQZ9V9Ml0/0.jpg)](https://www.youtube.com/watch?v=K7xQZ9V9Ml0)
```

Comment:
can you please also add:
https://streamnative.io/blog/tech/2020-02-04-how-to-build-database/
https://streamnative.io/blog/tech/2020-04-14-distributed-database-bk2/
https://streamnative.io/blog/tech/2020-05-12-distributed-database-bk3/

https://medium.com/streamnative/how-to-use-apache-pulsar-manager-with-herddb-dd265c955ca4

Reply:
Created wiki page with links to talks and deep dives and linked to the README: https://github.com/diennea/herddb/wiki/Talks-&-Publications